cpp/src/io/statistics/statistics_type_identification.cuh

```diff
@@ -133,13 +140,14 @@ class extrema_type {
   // decimal128        -> __int128_t
   // duration_[T]      -> int64_t
   // string_view       -> string_view
+  // byte_array_view   -> byte_array_view
   // timestamp_[T]     -> int64_t
 
  public:
   // Does type T have an extrema?
-  static constexpr bool is_supported = std::is_arithmetic_v<T> or std::is_same_v<T, string_view> or
-                                       cudf::is_duration<T>() or cudf::is_timestamp<T>() or
-                                       cudf::is_fixed_point<T>();
+  static constexpr bool is_supported =
```

Comment:
TIL. I've only ever written this sort of thing as a `constexpr` function. It never occurred to me to use a constant.